js/app/Data/BaseTiledFormat.js

```diff
@@ -267,7 +267,8 @@ define([
     search: function(query, offset, limit, cb) {
       var self = this;
 
-      var url = self.getUrl("search", data, -1) + "/search?query=" + encodeURIComponent(query);
+      var key = {query: query, offset: offset, limit: limit};
+      var url = self.getUrl("search", key, -1) + "/search?query=" + encodeURIComponent(query);
```

Comment:
@redhog I'm not that sure that here was when we broke stuff:
https://github.com/GlobalFishingWatch/pelagos-client/commit/fca7b291e2f695d9042098fbb8ab59513f4b13b4
We tested the search API after that changed and worked. Also, now that the search is done with `get` all parameters are sent on the URL that is why you deleted the `data` hash.